doc/devbox_setup.md

```diff
@@ -113,4 +113,4 @@ Run the following as admin: `iothub\service\tests\prerequisites\windows_install.
 [UWP]:https://msdn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide
 [.NET]:https://www.microsoft.com/net
 [UWP]:https://msdn.microsoft.com/en-us/windows/uwp/winrt-components/index
-[Xamarin]:https://www.xamarin.com/
\ No newline at end of file
+[Xamarin]:https://www.xamarin.com/
```

Comment:
Why do we add link to Xamarin?

Reply:
Nothing is actually changed there: not sure why GH considers that a change (maybe spacing?).